packages/ds-medicare-gov/src/styles/components/_Forms.scss

```diff
@@ -68,26 +51,4 @@ legend[id^='datefield_label'] {
       text-transform: none;
     }
   }
-
-  > .ds-c-field__hint:not(.ds-u-color--error) {
-    color: var(--form-label__color--not-error);
-  }
-}
-
-.ds-c-field__hint {
-  font-family: var(--font-family-rubik);
-  font-size: var(--font-size-sm);
-}
-
-.ds-c-datefield__separator {
-  display: none;
-}
-
-// Alignment of $ icon for the currency field
-.ds-c-field--currency {
-  padding-left: $spacer-3;
```

Comment:
The currency tokens in `medicare.ts`, lines 443-445 come from these styles. The spacer value had to be adjusted from the override so the "$" looked more correct with the masked field.

Reply:
I agree. @line47 or @brentschneider, please [take a look at the VRT reference image change](https://github.com/CMSgov/design-system/pull/2480/files#diff-b6ce95ded0ccb2b7c3464d1c64b3713a440a9b19913776ba5a2e49fca0d78a21) to see how it changed